keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/operators/binary/ArcTan2Vertex.java

```diff
@@ -10,16 +12,31 @@
 
 public class ArcTan2Vertex extends DoubleBinaryOpVertex {
 
+    private static final String X_NAME = LEFT_NAME;
+    private static final String Y_NAME = RIGHT_NAME;
+
     /**
      * Calculates the signed angle, in radians, between the positive x-axis and a ray to the point (x, y) from the origin
      *
      * @param x x coordinate
      * @param y y coordinate
      */
-    public ArcTan2Vertex(DoubleVertex x, DoubleVertex y) {
+    @ExportVertexToPythonBindings
+    public ArcTan2Vertex(@LoadParentVertex(X_NAME) DoubleVertex x,
+                         @LoadParentVertex(Y_NAME) DoubleVertex y) {
         super(x, y);
     }
 
+    // No SaveParentVertex annotation needed as DoubleBinaryOpVertex.getLeft() will be used when saving the vertex
```

Comment:
I don't think the comments are needed here - a bit overkill

Reply:
Cool - will remove